src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpCollectionExpressionRewriter.cs

```diff
@@ -61,6 +61,7 @@ internal static class CSharpCollectionExpressionRewriter
 #endif
 
         var initializer = getInitializer(expressionToReplace);
+        var endOfLine = EndOfLine(formattingOptions.NewLine);
```

Comment:
We were previously using elastic trivia, and depending on the formatter doing things right.  But i've foudn the formatter too finicky.  ANd we are otherwise fully controlling trivia in this rewriter.  So i just switched to making 100% of it explicit.  NO elastic trivia.